client/app/routes/notifications.tsx

```diff
@@ -0,0 +1,48 @@
+import { LoaderFunctionArgs, MetaFunction, json } from "@remix-run/node";
+import { NavLink, useLoaderData } from "@remix-run/react";
+import { checkAuth } from "~/lib/check-auth";
+import { prisma } from "~/lib/prisma.server";
+import { values } from "~/lib/values.server";
+
+export const loader = async ({ request }: LoaderFunctionArgs) => {
+	const userId = await checkAuth(request);
+	const notifications = await prisma.notificationSuscribers.findMany({
+		where: { userId, read: false },
+		include: { notification: true, user: true },
```

Comment:
Any reason for including `user`?

Reply:
Yeah I have to remove, initially thought i would get username to construct the message..but could not so its unnecessary.